simulation_parameters/common/wiki.json

```diff
@@ -48,7 +50,46 @@
           "SectionHeading": "WIKI_HEADING_SCIENTIFIC_BACKGROUND",
           "SectionBody": "WIKI_AXON_SCIENTIFIC_BACKGROUND"
         }
-      ]
+      ],
+      "InfoboxData": [
+        {
+          "InfoboxKey": "INFO_BOX_COST",
+          "InfoboxValue": "100"
```

Comment:
I think it would be better to reuse existing JSON keys that are translated as they fit well enough and that wouldn't require changing the translation extract. Something like `Name` should be fine and understandable. Perhaps `Description` could work for the value or maybe this could have a new property name that is added to the translation system, maybe something like `DisplayedValue`?